apps/meteor/app/lib/server/lib/notifyUsersOnMessage.ts

```diff
@@ -16,36 +19,36 @@ function messageContainsHighlight(message: IMessage, highlights: string[]): bool
 	});
 }
 
-export async function getMentions(message: IMessage): Promise<{ toAll: boolean; toHere: boolean; mentionIds: string[] }> {
-	const {
-		mentions,
-		u: { _id: senderId },
-	} = message;
-
-	if (!mentions) {
-		return {
-			toAll: false,
-			toHere: false,
-			mentionIds: [],
-		};
-	}
+export async function getMentions(message: IMessage): Promise<IMentionCounter> {
+	if (!message.mentions) return { toAll: false, toHere: false, mentionIds: [] };
+
+	let toAll = false;
+	let toHere = false;
+
+	const mentions = message.mentions.reduce(
+		(acc, cur) => {
+			if (cur._id === message.u._id) return acc;
```

Comment:
What about create a variable with the name of this condition? I'm not sure what does it mean 🤔 WDYT?